druid/src/widget/label.rs

```diff
@@ -30,10 +28,15 @@ use crate::localization::LocalizedString;
 use crate::theme;
 use crate::{Point, RenderContext};
 
-/// The text for the label; either a localized or a specific string.
+/// The text for the label
 pub enum LabelText<T> {
+    /// Localized string that will be resolved through `Env`.
     Localized(LocalizedString<T>),
+    /// Specific text
     Specific(String),
+    /// The provided closure is called on update, and its return
+    /// value is used as the text for the label.
+    Dynamic(Box<dyn Fn(&T, &Env) -> String>),
 }
 
 /// A label that displays some text.
```

Comment:
This would be a good time to plump up these docs a bit, showing the different ways the text can be provided?

Reply:
Added usage docs.